org-roam.el

```diff
@@ -441,6 +441,18 @@ Examples:
            (slug (-reduce-from #'cl-replace (strip-nonspacing-marks title) pairs)))
       (s-downcase slug))))
 
+;;;; Handler for template errors
+(defmacro org-roam--with-template-error (templates &rest body)
+  "Eval BODY, and point user to TEMPLATES on error.
+
+\(fn TEMPLATES BODY...)"
+  (declare (debug (form body)) (indent 1))
+  `(condition-case err
+       ,@body
+     (error (user-error "%s.  Please adjust `%s'"
+                        (error-message-string err)
+                        ,templates))))
+
```

Comment:
Should this live in org-roam-macs.el?

Reply:
Definitely.

Reply:
Done in2deb02fdfe6295fd3ddb85ad03353c7fe9d97e0c.